fuzzers/fuzzbench_forkserver/src/main.rs

```diff
@@ -346,7 +346,7 @@ fn fuzz(
         cmplog_shmem.write_to_env("__AFL_CMPLOG_SHM_ID").unwrap();
         let cmpmap = unsafe { cmplog_shmem.as_object_mut::<AFLppCmpMap>() };
 
-        let cmplog_observer = StdCmpObserver::new("cmplog", cmpmap, true);
+        let cmplog_observer = ForkserverCmpObserver::new("cmplog", cmpmap, true);
```

Comment:
If we need different observers for different executors, can we add type constraints so they cannot be misused?

Reply:
for this ForkserverCmpObserver, No.

I think this is related issue to https://github.com/AFLplusplus/LibAFL/issues/23.
I think we have too many *hidden* dependency
"module this depends on module that" but that is not very clear if the user understands well about each module.